packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -79,6 +79,24 @@
   void onError(String errorCode, String errorMessage);
 }
 
+/** A mockable wrapper for CameraDevice calls. */
+interface CameraDeviceWrapper {
+  @NonNull
+  CaptureRequest.Builder createCaptureRequest(int var1) throws CameraAccessException;
```

Comment:
var1 doesn't seem to provide meaning. Did you mean templateType?

Reply:
Oops! The declarations of these methods didn't have useful variable names for some reason. I thought I fixed them all when I copied them.